src/Provider/GeocodeEarth/README.md

```diff
@@ -8,7 +8,56 @@
 [![Software License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE)
 
 This is the Geocode Earth provider from the PHP Geocoder. This is a **READ ONLY** repository. See the
-[main repo](https://github.com/geocoder-php/Geocoder) for information and documentation. 
+[main repo](https://github.com/geocoder-php/Geocoder) for information and documentation.
+
+## Usage
+
+```php
+use Http\Adapter\Guzzle7\Client as GuzzleAdapter;
+use Geocoder\Provider\GeocodeEarth\GeocodeEarth;
+use Geocoder\Query\GeocodeQuery;
+use Geocoder\Query\ReverseQuery;
+
+// Create a new HTTP adapter
+$httpClient = new GuzzleAdapter();
+
+// You must provide an API key
+$provider = new GeocodeEarth($httpClient, 'your-api-key');
+
+// Forward geocode
+$forward = $provider->geocodeQuery(GeocodeQuery::create('11 Wall Street Manhattan')->withLimit(1));
+print_locations($forward);
+
+// Reverse geocode
+$reverse = $provider->reverseQuery(ReverseQuery::fromCoordinates(40.707141, -74.010865)->withLimit(1));
+print_locations($reverse);
+
+// Print locations to console
+function print_locations($results) {
+  foreach($results as $key => $location) {
+    printf(
+      "%d) [%.6f, %0.6f] %s %s, %s, %s %s, %s\n",
+      $key,
+      $location->getCoordinates()->getLatitude(),
+      $location->getCoordinates()->getLongitude(),
+      $location->getStreetNumber(),
+      $location->getStreetName(),
+      $location->getSubLocality(),
+      $location->getPostalCode(),
+      $location->getLocality(),
+      $location->getCountry()
+    );
+  }
+}
+```
+
+```bash
+0) [40.707141, -74.010865] 11 Wall Street, Financial District, 10005 New York, United States
+0) [40.707141, -74.010865] 11 Wall Street, Financial District, 10005 New York, United States
+```
```

Comment:
To be honest, I don't really see the point of this part.
Usage is described in the [main README](https://github.com/geocoder-php/Geocoder/#usage) and the [documentation](https://geocoder-php.org/docs/#usage).

Reply:
This was based on the [Google Maps Provider](https://github.com/geocoder-php/Geocoder/tree/master/src/Provider/GoogleMaps) which provides a 'Usage' section.

Reply:
Oh okay, I see 😄 

I'll create another PR for the README files of each provider to make it more consistent (full example in main README and maybe a link from provider README to main README and small example with just the constructor in provider README).